gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -66,10 +66,6 @@
  */
 @javax.annotation.Generated("by GAPIC")
 public class PublisherApi implements AutoCloseable {
```

Comment:
Garrett maybe a better person than me to answer this question. @garrettjonesgoogle